lib/main_common.pm

```diff
@@ -1433,6 +1424,10 @@ sub load_extra_tests_desktop {
         loadtest "x11/gnucash";
 
     }
+    # https://progress.opensuse.org/issues/37342
+    if (is_sle() && gnomestep_is_applicable() && !is_staging()) {
+        loadtest "x11/remote_desktop/vino_screensharing_available";
```

Comment:
@pdostal any reason why this should not be scheduled on openSUSE?

Reply:
No, there's a bug right now, but it will be resolved soon. After this is resolved for SLE my plan is to include it in openSUSE too.

Reply:
Makes sense, thank you. Then this should not block this PR.